R/stat_summary.R

```diff
@@ -11,6 +11,29 @@
 #'
 #'@examples
 #'stat_summary <- function(pop, samples, parameter)
+#'
+#' @importFrom rlang :=
+#' @export
 stat_summary <- function(population, samples, parameter) {
 
+  sub_pop <- population[[names(population)]]
+  sub_samples <- samples[[names(samples)]]
+
+  pop <- dplyr::tibble('data' := c('population', 'samples'))
+
+  for (i in (1:length(parameter))) {
+    if (class(try(match.fun(parameter[i]), silent = TRUE)) == "try-error") {
+
+      stop('This is not a vaild function')
+
+    } else {
+      para_func <- match.fun(parameter[i])
+    }
```

Comment:
```suggestion
  for (i in (1:length(parameter))) {
    if (class(try(match.fun(parameter[i]), silent = TRUE)) == "try-error") {
      stop('This is not a vaild function')
    } else {
      para_func <- match.fun(parameter[i])
    }
```

spacing to match rest of code

Reply:
Great point! I will check both of my functions. Addressed #40